packages/quickwit/v0.8.1+3/package.yaml

```diff
@@ -2,6 +2,9 @@
 name: quickwit
 shortDescription: Cloud-native search engine for observability
 longDescription: |
+  > _**PLEASE READ:** If you previously installed this package as a `ClusterPackage`, please remove and install it as a
```

Comment:
wouldn't it make sens to include that in the older versions too? because if someone has an old version installed, they see the package description of the installed older version.